apps/admin-ui/src/user/UserAttributes.tsx

```diff
@@ -41,7 +45,10 @@ export const UserAttributes = ({ user: defaultUser }: UserAttributesProps) => {
 
   const save = async (attributeForm: AttributeForm) => {
     try {
-      const attributes = keyValueToArray(attributeForm.attributes!);
+      const attributes = Object.assign(
+        user.attributes || {},
+        keyValueToArray(attributeForm.attributes!)
+      );
```

Comment:
`Object.assign()` will mutate the original reference (in this case `user.attributes`), you might want to provide an empty object as the first argument to prevent this.
```suggestion
      const attributes = Object.assign(
        {},
        user.attributes || {},
        keyValueToArray(attributeForm.attributes!)
      );
```